warehouse/query-core/src/main/java/datawave/query/jexl/visitors/TreeHashVisitor.java

```diff
@@ -141,14 +154,26 @@ public Object visit(ASTAndNode node, Object data) {
         TreeHashNode inte = (TreeHashNode) (data);
         int lastsize = inte.length();
         
-        for (int i = 0; i < numChildren; i++) {
+        // Build keys for child node map
```

Comment:
can you pull this out as a function and reuse it? Looks copy/pasted

Reply:
Yes, refactored in next commit.